bundles/org.openhab.binding.groheondus/src/main/java/org/openhab/binding/groheondus/internal/handler/GroheOndusAccountHandler.java

```diff
@@ -142,7 +142,9 @@ public void initialize() {
                 ondusService = OndusService.login(storage.get(STORAGE_KEY_REFRESH_TOKEN));
                 scheduleTokenRefresh();
             } else {
-                ondusService = OndusService.login(config.username, config.password);
+                // TODO: That's probably really inefficient, internally the loginWebform method acquires a refresh
+                // token, maybe there should be a way to obtain this token here, somehow.
+                ondusService = OndusService.loginWebform(config.username, config.password);
```

Comment:
This change is functionally equivalent though right?

Reply:
Yes, it just uses another method to login :) From a user point of view, nothing really changes with that.